ports/zephyr/dynamic_pinctrl/rp2.h

```diff
@@ -0,0 +1,36 @@
+/* Dynamic Pinctrl assembly functions file.
```

Comment:
How about "assembly functions" -> "builder functions" to avoid mistaking for assembly language?
But it's quick to see that this is not about assembly language though...

Reply:
done